File: tabuada.py
from numpy import integer


class tabuada:
    def exibirTabuada(valor):
        numero = valor
        for num in range(11):
            print(f'{numero} x {num} = {numero*num}')
    
    def solicitarNumero():
        pass


if __name__=='__main__':
    print("Deseja a tabuada para qual valor: ")
    numero = int(input())
    tabuada.exibirTabuada(numero)

File: Tabuada.java
import java.util.Scanner;

public class Tabuada {
    public static void exibirTabuada(int valor){
        for(int i = 1; i < 11; i++){
            System.out.println(valor+" * "+ i +" = " + (i*valor));
        }
    }
    public static void main(String[]args){
        Scanner s = new Scanner(System.in);
        System.out.println("Deseja a tabuada para qual valor: ");
        int numero = Integer.parseInt(s.next());
        Tabuada.exibirTabuada(numero);
        
        s.close();
    }
}
